src/main/kotlin/no/nav/syfo/db/AltinnLpsOppfolgingsplanDao.kt

```diff
@@ -0,0 +1,249 @@
+            it.setTimestamp(2, Timestamp.valueOf(LocalDateTime.now()))
+            it.setObject(3, uuid)
+            it.executeUpdate()
+        }
+        connection.commit()
+        rowsUpdated
+    }
+}
+
+fun DatabaseInterface.updateJournalpostId(uuid: UUID, journalpostId: String): Int {
+    val updateStatement = """
+        UPDATE ALTINN_LPS
+        SET journalpost_id = ?, last_changed = ?
+        WHERE uuid = ?
+    """.trimIndent()
+
+    return connection.use { connection ->
+        val rowsUpdated = connection.prepareStatement(updateStatement).use {
+            it.setString(1, journalpostId)
+            it.setTimestamp(2, Timestamp.valueOf(LocalDateTime.now()))
+            it.setObject(3, uuid)
+            it.executeUpdate()
+        }
+        connection.commit()
+        rowsUpdated
+    }
+}
+
+fun DatabaseInterface.setSentToNavTrue(uuid: UUID): Int {
+    val updateStatement = """
+        UPDATE ALTINN_LPS
+        SET sent_to_nav = TRUE, last_changed = ?
+        WHERE uuid = ?
+    """.trimIndent()
+
+    return connection.use { connection ->
+        val rowsUpdated = connection.prepareStatement(updateStatement).use {
+            it.setTimestamp(1, Timestamp.valueOf(LocalDateTime.now()))
+            it.setObject(2, uuid)
+            it.executeUpdate()
+        }
+        connection.commit()
+        rowsUpdated
+    }
+}
+
+fun DatabaseInterface.setSentToFastlegeTrue(uuid: UUID): Int {
+    val updateStatement = """
+        UPDATE ALTINN_LPS
+        SET sent_to_fastlege = TRUE, last_changed = ?
+        WHERE uuid = ?
+    """.trimIndent()
+
+    return connection.use { connection ->
+        val rowsUpdated = connection.prepareStatement(updateStatement).use {
+            it.setTimestamp(1, Timestamp.valueOf(LocalDateTime.now()))
+            it.setObject(2, uuid)
+            it.executeUpdate()
+        }
+        connection.commit()
+        rowsUpdated
+    }
+}
+
+fun DatabaseInterface.updateSendToFastlegeRetryCount(uuid: UUID, prevCount: Int): Int {
+    val updateStatement = """
+        UPDATE ALTINN_LPS
+        SET send_to_fastlege_retry_count = ?, last_changed = ?
+        WHERE uuid = ?
+    """.trimIndent()
+
+    return connection.use { connection ->
+        val rowsUpdated = connection.prepareStatement(updateStatement).use {
+            it.setInt(1, prevCount + 1)
+            it.setTimestamp(2, Timestamp.valueOf(LocalDateTime.now()))
+            it.setObject(3, uuid)
+            it.executeUpdate()
+        }
+        connection.commit()
+        rowsUpdated
+    }
+}
+
+fun DatabaseInterface.getAltinnLpsOppfolgingsplanByUuid(lpsUUID: UUID): AltinnLpsOppfolgingsplan {
+    val queryStatement = """
+        SELECT *
+        FROM ALTINN_LPS
+        WHERE uuid = ?
+    """.trimIndent()
+
+    return connection.use { connection ->
+        connection.prepareStatement(queryStatement).use {
+            it.setObject(1, lpsUUID)
+            it.executeQuery().toList { toAltinnLpsOppfolgingsplan() }
```

Comment:
Hm, trenger man å først gjøre det om til liste for å så hente ut første elementet, når man spør etter 1 spesifikk uuid?

Reply:
@AudunSorheim Det er sant 👍 Lagde en metode for å hente ut et enkelt object fra et ResultSet (`toObject`) i `no.nav.syfo.db.Util`